packages/account/src/Components/poi/status/unsupported/card-details/uploader.tsx

```diff
@@ -124,36 +143,29 @@ const Uploader = ({ data, value, is_full, onChange, has_frame }) => {
                         filename_limit={32}
                         hover_message={localize('Drop files here..')}
                         max_size={max_document_size}
-                        message={open => <Message open={open} data={data} />}
+                        message={(open?: () => void) => <Message open={open} data={data} />}
                         preview_single={
                             image && (
                                 <Preview
                                     data={data}
                                     value={image}
                                     has_frame={has_frame}
                                     setFieldValue={setFieldValue}
-                                    handleChange={handleChange}
+                                    handleChange={() => handleChange()}
                                 />
                             )
                         }
                         multiple={false}
-                        onDropAccepted={files => handleAccept(files, setFieldValue)}
-                        onDropRejected={files => handleReject(files, setFieldValue)}
-                        validation_error_message={value?.errors?.length ? ValidationErrorMessage : null}
+                        onDropAccepted={(files: object[]) => handleAccept(files, setFieldValue)}
+                        onDropRejected={(files: Array<object>) => handleReject(files, setFieldValue)}
+                        validation_error_message={value?.errors?.length ? ValidationErrorMessage : undefined}
                         noClick
-                        value={image ? [image] : []}
+                        value={(image ? [image] : []) as unknown as React.ComponentProps<typeof FileDropzone>['value']}
```

Comment:
I couldn't find another way to solve this TS migration better. If you have any ways on doing this, please let me know 🙏